calendar/calendar-impl/impl/src/java/org/sakaiproject/calendar/impl/BaseCalendarService.java

```diff
@@ -142,6 +142,8 @@
 	
 	private Cache cache = null;
 	
+	private String m_calRefList = "";
```

Comment:
Yeah, thanks, Earle already explained me that. I have thinking how could I fix this and I think that the best choice would be to use a ConcurrentHashMap in BaseExternalCalendarSubscriptionService.java to store the references.

If you know a better solution I am willing to know it.